### DISCUSSION TITLE
Martingale Bet

### DISCUSSION THREAD

**Discussion by ghost** [lmusarella/Pancakeswap-Prediction-Bot-2023#18] [Ideas]
Now the Martingale strategy waits to bet until the pending round is finished, this is understandable because otherwise it can;t determine win/loss.
So there is always a round in between, can you extend the bot to "simply" bet on that round in between with the "default" bet amount on the same side as the pending round.

**Comment by tankiatwee** [comment]
Try running 2 instances (or windows) of the node.js command prompt, 1 running odd rounds and the other running even rounds.
